Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -0,0 +1,143 @@
+          .foregroundStyle(Color(braveSystemName: .textInteractive))
+        Text(Strings.Shields.reportBrokenSiteBody2)
+          .font(.footnote)
+        BraveTextEditor(
+          text: $additionalDetails,
+          prompt: Strings.Shields.reportBrokenAdditionalDetails
+        ).frame(height: 80)
+        
+        VStack(alignment: .leading, spacing: 4) {
+          Text(Strings.Shields.reportBrokenContactMe).font(.caption)
+          TextField(
+            Strings.Shields.reportBrokenContactMe,
+            text: $contactDetails, prompt: Text(Strings.Shields.reportBrokenContactMeSuggestions)
+          )
+          .textContentType(.emailAddress)
+          .textFieldStyle(BraveTextFieldStyle())
+          .autocorrectionDisabled()
+          .textInputAutocapitalization(.never)
+        }
+      }
+      .padding()
+      .foregroundStyle(Color(braveSystemName: .textSecondary))
+    }
+    .background(Color(.braveBackground))
+    .foregroundStyle(Color(braveSystemName: .textSecondary))
+    .navigationTitle(Strings.Shields.reportABrokenSite)
+    .toolbar {
+      ToolbarItem(placement: .cancellationAction) {
+        Button(Strings.cancelButtonTitle) {
+          dismiss()
+        }
+        .disabled(isSubmittingReport)
+        .hidden(isHidden: isSubmitted)
+        .animation(.smooth.speed(0.2), value: isSubmitted)
+      }
+      
+      ToolbarItem(placement: .confirmationAction) {
+        if isSubmitted {
+          Button(Strings.close, action: {
+            dismiss()
+          })
+        } else {
+          Button(Strings.Shields.reportBrokenSubmitButtonTitle, action: {
+            withAnimation {
+              isSubmittingReport = true
+            }
+            
+            Task { @MainActor in
+              await createAndSubmitReport()
+              
+              withAnimation {
+                isSubmitted = true
+              }
+              
+              try await Task.sleep(seconds: 4)
+              dismiss()
+            }
+          }).disabled(isSubmittingReport)
+        }
+      }
+    }
+    .overlay {
+      if isSubmittingReport {
+        ProgressView()
+          .progressViewStyle(.braveCircular(size: .normal, tint: .braveBlurpleTint))
+          .frame(maxWidth: .infinity, maxHeight: .infinity)
+          .background(Color(.braveBackground).opacity(0.5).ignoresSafeArea())
+          .animation(.easeInOut, value: isSubmittingReport)
+      }
+    }
+    .overlay {
+      if isSubmitted {
+        SubmitReportSuccessView()
+          .progressViewStyle(.braveCircular(size: .normal, tint: .braveBlurpleTint))
+          .frame(maxWidth: .infinity, maxHeight: .infinity)
+          .background(Color(.braveBackground).ignoresSafeArea())
+          .animation(.easeIn, value: isSubmitted)
+      }
+    }
+  }
+  
+  var body: some View {
+    NavigationView {
+      scrollContent
+        .navigationBarTitleDisplayMode(.inline)
+        .navigationViewStyle(.stack)
```

Comment:
Needs to go on the `NavigationView` not the content inside

Reply:
Ahh sorry I should have realized this